### ISSUE TITLE
--reverse not working anymore in 0.10.0 ?

### ISSUE THREAD

**Issue by hnb2** [naiquevin/pipdeptree#73]
Hi there,

I just upgraded from 0.9.0 to 0.10.0 and I noticed that the output of my usual command is different:

## 0.9.0

```bash
pipdeptree --packages mando --reverse

mando==0.3.3
  - radon==1.5.0 [requires: mando<0.4,>=0.3]
```

## 0.10.0

```bash
pipdeptree --packages mando --reverse

mando==0.3.3
```

Example project: https://pypi.python.org/pypi/radon

I tried with both `-r` and `--reverse`, same problem, I think something broke.

**Comment by naiquevin** [comment]
Yes, the behaviour has indeed changed. Missed this when reviewing the PR. I am fixing it.

**Comment by naiquevin** [comment]
I've released ```0.10.1``` on PyPI to fix this. Thanks for reporting.
